### DISCUSSION TITLE
fix error lib

### DISCUSSION THREAD

**Discussion by farshad-fallah** [BearWare/TeamTalk5#2023] [Q&A]
hello how i can fix it thanks befor
./tt5srv: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by ./tt5srv)
./tt5srv: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by ./tt5srv)
./tt5srv: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by ./tt5srv)

**Comment by bear101** [comment]
it looks like you're trying to run the TeamTalk server on a very old Linux distribution.

**Reply by farshad-fallah** [reply]
thanks a lot i upgrade to ver 2204 and my problem is solved
